README.md

```diff
@@ -72,6 +72,9 @@ Please always make the Pull Request against the develop branch.
     
 * Thank you for your contribution!
 
+> In development mode, you will certainly have to enable [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) on your browser.
+> The Cross-origin resource sharing of browsers block every AJAX request that not match the exact host, protocol, and port of your site. 
```

Comment:
"The Cross-origin resource sharing of browsers block every AJAX request that not match the exact host, protocol, and port of your site." --> "The Cross-Origin Resource Sharing (CORS) performed by the browser blocks every AJAX request that does not match the exact host, protocol, and port of your site." ?

Reply:
better yes